### DISCUSSION TITLE
Did shakapacker remove erb support?

### DISCUSSION THREAD

**Discussion by RailsCod3rFuture** [shakacode/shakapacker#340] [General]
I'm currently trying to retrieve js thats inside of a gem, but the command bundle exec rails shakapacker:install:erb is not working. Is there an alternative way to get the js from inside of a local gem?

**Comment by ahangarha** [comment]
The command you refer to belongs to `webpacker`, not `shakapacker`.
Shakapacker doesn't support such install commands.

May you elaborate further on what your use case is?
